blog/2010/10/clarifications-about-call-for-instructor-training.html

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "Clarification about December Instructor Training"
+date: 2015-11-06
+time: "22:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+Since we announce
+<a href="{{page.root}}/blog/2015/10/call-for-instructor-training.html">that the applications for December Instructor Training</a>
+we receive a lot of questions. We amend the announcement with the answer to some
+questions and we believe that others need a post dedicated to them.
```

Comment:
Since we announced <link> that we were taking applications for the December Instructor Training, we have received a lot of questions. We are now amending the announcement to clarify some of the things that have come up.